nextflow.config

```diff
@@ -430,8 +430,8 @@ params {
 
     r_contaminants {
         // container contains minimap2 and samtools
-        singularity = "https://depot.galaxyproject.org/singularity/mulled-v2-66534bcbb7031a148b13e2ad42583020b9cd25c4:1679e915ddb9d6b4abda91880c4b48857d471bd8-0"
-        docker = "quay.io/biocontainers/mulled-v2-66534bcbb7031a148b13e2ad42583020b9cd25c4:8f2087d838e5270cd83b5a016667234429f16eea-0"
+        singularity = "https://depot.galaxyproject.org/singularity/mulled-v2-66534bcbb7031a148b13e2ad42583020b9cd25c4:3161f532a5ea6f1dec9be5667c9efc2afdac6104-0"
```

Comment:
Since this PR involves updating some containers, can you add this change to the `CHANGELOG.md` (and also add a message about the existing change you've made on modifying the qc messages to the changelog)?

Reply:
thank you for reminding me!

Reply:
fixed in:  https://github.com/phac-nml/mikrokondo/pull/85/commits/397fa76a0aec1dfd5898c41ae0a8bf18cb496d64